docs/use/originating-assets/index.md

```diff
@@ -0,0 +1,82 @@
+---
+id: originating-assets
+order: 10
+title: Originating Assets 
+---
+## High level user flow
+![](https://storage.googleapis.com/centrifuge-hackmd/upload_9e677fa9504dc2cdfae62dd3b6d1fcab.png)
+
+## Mint NFT in Gateway
+### What is the Gateway
+[Centrifuge Gateway](https://gateway.centrifuge.io/) is the GUI of a Centrifuge Node that allows to securely create and exchange customised documents representing real-world as part of the Centrifuge P2P-protocol and mint NFTs based on these documents.
```

Comment:
@annamehr , the part "real-world" is not clear here: is it an adjective, or a nound? I'm suggesting (if noun) "representing the real world", or (if adjective) "representing real-world collateral"...